### ISSUE TITLE
Screen reader does not announce search suggestion information after entering data in the 'Message Input Box' edit field present in the dialog.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5104]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

User who rely on screen readers are getting impacted if Screen reader does not announce searching result after entering data in the 'Message Input Box' present in the dialog, the screen reader user will not be able to get proper information on these controls and will be left unaware about the visual information. This will lead to distortion.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

Request Contributors - POC (sharepoint.com)

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers with valid credentials.
Step 2: Invoke the 'Ask iris now' button which is present under the questions section.
Iris the bot dialog will appear. Type “How do I find” in type your message 'Message Input Box' edit field.
Step 3: Verify that Screen reader does not announce screen reader does not announce search suggestion information after entering data in the 'Message Input Box' edit field present in the dialog or not.

### What do you expect?

Screen reader should announce search suggestion information after entering data in the 'Message Input Box' edit field present in the dialog.

### What actually happened?

Screen reader does not announce search suggestion information after entering data in the 'Message Input Box' edit field present in the dialog.
Observation: When focus lands on message input box edit field, after entering data screen reader is not announcing any information.

### Do you have any screenshots or recordings to repro the issue?

![MAS 1 3 1_Screen reader does not announce search suggestion information after entering data in the 'Message Input Box' edit field present in the dialog](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/2630ec05-3d02-41b4-8c8a-55308a35d5b2)

### MAS reference

https://aka.ms/MAS1.3.1

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
This is outside of the scope of Web Chat. The auto-complete is a customization done by Iris Bot and this is not implemented in Web Chat.

**Comment by Kirank915** [comment]
As per the above comment, this issue is not under bot framework team. Hence closed the bug and partner team (WWL) also changed the ownership in bugs.
